hicommonwealth/commonwealth
# [Admin-Onboarding] New Community Profile Screen

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6024

## Description of Changes
- Adds FIXME widget to TODO page.
- 

## "How We Fixed It"


## Test Plan
- Enable the `FLAG_NEW_ADMIN_ONBOARDING` flag
- Go to any community where you are an admin
- Go to the manage/profile page
- Verify that the page is responsive on different screen sizes
- CA and Verify that the Community Profile form is working as expected and without any issues.
- Update community name: press the lock icon in the field to enable it and enter the community name (when existing and new community names don't match, verify that pressing the lock icon again is disabled).
- Verify community image upload: Upload a new community image and verify it works. Also, try to generate an AI image and verify it works.
- Verify that changing community tags, banner, and default page settings refreshes the page correctly and the banner is made visible whenever the banner text is present.
- Verify that custom stages input accepts a valid input pattern.
- Verify that there aren't any other issues.

## Deployment Plan
N/A

## Other Considerations
N/A
